cmd/maya-apiserver/app/server/volume_endpoint_v1alpha1.go

```diff
@@ -32,81 +30,208 @@ func (s *HTTPServer) volumeV1alpha1SpecificRequest(resp http.ResponseWriter, req
 	}
 
 	switch req.Method {
-	case "PUT", "POST":
+	case "POST":
 		return volOp.create()
 	case "GET":
-		return volOp.get()
+		return volOp.httpGet()
+	case "DELETE":
+		return volOp.httpDelete()
```

Comment:
This `GET` and `DELETE` cases will not be hit as we enter this handler with http method of `POST` and `PUT` only.
This is what I mentioned in above comment also wrt `volumeV1alpha1SpecificRequest` handler.

Reply:
Yes. You are right. This is the intention in this PR.